opentech/apply/projects/templates/application_projects/project_simplified_detail.html

```diff
@@ -0,0 +1,81 @@
+{% extends "base-apply.html" %}
+
+{% block title %}{{ object.title }}{% endblock %}
+
+{% block body_class %}{% endblock %}
+
+{% block content %}
+<div class="admin-bar">
+    <div class="admin-bar__inner">
+        <a href="{% object.get_absolute_url %}">Project</a>
+        <h1 class="beta heading heading--no-margin heading--bold">{{ object.title }}</h1>
+        <h5 class="heading heading--meta">
+            <span>{{ object.submission.page }}</span>
+            <span>{{ object.submission.round }}</span>
+            <span>Lead: {{ object.lead }}</span>
+        </h5>
+    </div>
+</div>
+
+<div class="wrapper wrapper--large">
+
+    {% block notifications %}
+    {% endblock %}
+    <div class="wrapper wrapper--sidebar">
+        <article class="wrapper--sidebar--inner">
+            <h3>Project Information</h3>
+            <div class="grid grid--proposal-info">
+                <div>
+                    <h5>Proposed start date</h5>
+                    <p>{{ object.proposed_start|date:"j F Y"|default:"-" }}</p>
+                </div>
+
+                <div>
+                    <h5>Project Proposed end date</h5>
+                    <p>{{ object.proposed_end|date:"j F Y"|default:"-" }}</p>
+                </div>
+
+                <div>
+                    <h5>Legal name</h5>
+                    <p>{{ object.contact_legal_name|default:"-" }}</p>
+                </div>
+
+                <div>
+                    <h5>Email</h5>
+                    <p>{{ object.contact_email|default:"-" }}</p>
+                </div>
+
+                <div>
+                    <h5>Address</h5>
+                    <p>{{ object.get_address_display|default:"-"}}</p>
+                </div>
+
+                <div>
+                    <h5>Phone</h5>
+                    <p>{{ object.phone|default:"-" }}</p>
+                </div>
+
+                <div>
+                    <h5>Value</h5>
+                    <p>${{ object.value|default:"-" }}</p>
+                </div>
+            </div>
+
+            {# TODO: put outcome of #1364 here #}
+
+            <div>
+                <p><a href="{% url 'apply:submissions:simplified' pk=object.submission_id %}">Submission</a></p>
+            </div>
+
+            <div>
+                <h5>Supporting Documents</h5>
+                {% for packet_file in object.packet_files.all %}
+                <p><a href="{{ packet_file.document.url }}">{{ packet_file.title }}</a></p>
```

Comment:
I think this needs to the `{% url 'apply:projects:document' ... %}` so that we are correctly using the private view.

Reply:
Fixed and rebased.